src/app/descriptors/org/questions.json

```diff
@@ -77,7 +77,7 @@
   },
   {
     "text": "כלל המכרזים והפטורים ממכרז עם הארגון",
-    "query": ["select tender_type as \"סוג הליך הרכש\",",
```

Comment:
okay, so I checked and there is already an existing column named `tender_type_he`...